test/universal.js

```diff
@@ -449,6 +449,17 @@ describe('Google Analytics :: Universal', function() {
       });
     });
 
+    describe('#refundedOrderPartial', function() {
+      it('should send the right data', function(done) {
+        var json = test.fixture('refunded-order-partial');
+        test
+          .set(settings)
+          .track(json.input)
+          .sendsAlmost(json.output, {ignored: ['qt']})
```

Comment:
Is this deployable now? Motley Fool waiting on it.

On Mon, Aug 22, 2016 at 11:57 AM, Joe Gershenson <notifications@github.com>
wrote:

> In test/universal.js
> <https://github.com/segment-integrations/integration-google-analytics/pull/48#discussion_r75737193>
> :
>
> > @@ -449,6 +449,17 @@ describe('Google Analytics :: Universal', function() {
> >        });
> >      });
> >
> > +    describe('#refundedOrderPartial', function() {
> > +      it('should send the right data', function(done) {
> > +        var json = test.fixture('refunded-order-partial');
> > +        test
> > +          .set(settings)
> > +          .track(json.input)
> > +          .sendsAlmost(json.output, {ignored: ['qt']})
>
> Pretty sure I wrote the sendsAlmost test to ignore qt for this reason.
> IIRC it's because it varies depending on exactly how fast the test runs.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/segment-integrations/integration-google-analytics/pull/48/files/2771b369518609a1538b4d34f07b1da1688abf67#r75737193>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF-F7HuX4ADnsIQw7mdAt4vDBpsXKXJFks5qifE0gaJpZM4Jo8Ac>
> .
>